java/code/src/com/redhat/rhn/domain/server/Server.java

```diff
@@ -499,7 +499,7 @@ public void storeConfigChannels() {
     /**
      * Protected constructor
      */
-    protected Server() {
+    public Server() {
```

Comment:
when I see something like "protected" I always think it might have a reason for having it.
Are we sure we need it public?

Reply:
I had some permission error, but it's better I'll investigate deeply on it

Reply:
https://github.com/uyuni-project/uyuni/pull/9682/files#r1952449489

Reply:
Ok